### PR TITLE
Itm 570 delegation dre (ADEPT UPDATES)

### PR DESCRIPTION
**Pull request by kaitlyn-sharo**
- Added some hard coded checks to update some text adept wanted updated
- Updated patient data to show up pretty in dashboard
- Fixed a few other adept problems
- Removed TAD-baseline
- Added quiet mode to speed up update_survey_config (will still print warnings)
- Added hard-coded adept supplies so that they are correct even for scenarios that start mid-yaml

To run, 
`cd delegation_survey`
`python3 dre_adms_for_delegation.py` (will take awhile)
`python3 update_survey_config.py`
Press Enter
Press 3
Press y
Press y

### CODE REVIEW

**Review by dereknop — APPROVED** [review]

**Review by phile-caci — APPROVED** [review]
No issues running, LGTM!

**Review by nextcen-dgemoets — APPROVED** [review]
The scripts ran cleanly, including after the script to remove duplicates.

**Comment by kaitlyn-sharo** [comment]
Waiting for Brian's script to be merged in before merging

**Pull request merged**
